doc/faq.rst

```diff
@@ -342,6 +342,13 @@ For bug reports or feature requests, please make use of the
   Please do not email any authors directly to ask for assistance, report bugs,
   or for any other issue related to scikit-learn.
 
+How can I obtain permission to use the images in scikit-learn for my work?
```

Comment:
I would put this entry in the "About this project" (first section).

I think we need to be more precise here. We are speaking about the images contained in the scikit-learn repository or generated in the scikit-learn documentation.